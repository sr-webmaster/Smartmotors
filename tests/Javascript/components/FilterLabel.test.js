import { createLocalVue, mount } from '@vue/test-utils'
import Component from '~/components/FilterLabel.vue'

describe('FilterLabel', () => {
  // Component instance "under test"
  let localVue
  describe('Snapshots', () => {
    test('defaults', () => {
      localVue = createLocalVue()
      const wrapper = mount(Component, {
        localVue: localVue
      })
      expect(wrapper.element).toMatchSnapshot()
    })
    test('color is black', () => {
      localVue = createLocalVue()
      const wrapper = mount(Component, {
        localVue: localVue,
        propsData: {
          color: 'black'
        }
      })
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
