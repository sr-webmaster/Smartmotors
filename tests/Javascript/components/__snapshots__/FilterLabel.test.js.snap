// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterLabel Snapshots color is black 1`] = `
<div
  class="black--text"
>
  <strong
    class="no-wrap"
  />
</div>
`;

exports[`FilterLabel Snapshots defaults 1`] = `
<div
  class="white--text"
>
  <strong
    class="no-wrap"
  />
</div>
`;
